op-node/flags/flags.go

```diff
@@ -182,8 +182,12 @@ var (
 	L1CacheSize = &cli.UintFlag{
 		Name: "l1.cache-size",
 		Usage: "Cache size for blocks, receipts and transactions. " +
-			"It's optional and a sane default of 3/2 the sequencing window size is used if this field is set to 0.",
+			"If this flag is set to 0, 2/3 of the sequencing window size is used. " +
```

Comment:
The more I think about it, the more I'm thinking that we could just remove this behavior from the L1Client setup. The cache size is so individual per network, so maybe trying to be smart here is just complicating things.